src/NSwag.SwaggerGeneration.WebApi/OperationResponseProcessor.cs

```diff
@@ -0,0 +1,94 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="OperationResponseProcessor.cs" company="NSwag">
+//     Copyright (c) Rico Suter. All rights reserved.
+// </copyright>
+// <license>https://github.com/NSwag/NSwag/blob/master/LICENSE.md</license>
+// <author>Rico Suter, mail@rsuter.com</author>
+//-----------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using NJsonSchema;
+using NJsonSchema.Infrastructure;
+using NSwag.SwaggerGeneration.Processors;
+using NSwag.SwaggerGeneration.Processors.Contexts;
+using NSwag.SwaggerGeneration.WebApi.Processors.Models;
+
+namespace NSwag.SwaggerGeneration.WebApi.Processors
+{
+    /// <summary>Generates the operation's response objects based on reflection and the ResponseTypeAttribute, SwaggerResponseAttribute and ProducesResponseTypeAttribute attributes.</summary>
+    public class OperationResponseProcessor : IOperationProcessor
```

Comment:
Most of the code from this type went in to a `SwaggerResponseBuilder` type which builds the response from attributes. I share the type between the two projects. Let me know if you have any concerns or suggestions on your preferred way of organizing this.